### ISSUE TITLE
How to evaluate detection results in my desktop rather than submit to the test server?

### ISSUE THREAD

**Issue by baiyancheng20** [zhaoweicai/mscnn#22]
HI, I divide the training dataset into training and val datasets and use the training dataset to train my detector. How do I test the detection results on my desktop? Do I use 'evaluate_object.cpp' to test my detector performance on the val dataset? I want to find good parameters and then submit my detector's results. Thank you!

**Comment by zhaoweicai** [comment]
I use `evaluate_object.cpp` to evaluate the detection performance. But the default code is for evaluation on test set. You need change the code a little bit to make it evaluate on val set.

**Comment by jinuhjoo** [comment]
Hi @zhaoweicai 
Could you specify in detail on how you altered and used evaluate_object.cpp
That is if I need to run this on my validation set, do I need to split the ground truth txt files in label_2?
Could you share your evaluate_object.cpp ?

Thanks ~~

**Comment by zhaoweicai** [comment]
hi @jinuhjoo 
Send me an email, I can share the evaluation code to you.

**Comment by jk1323** [comment]
I have a same question as jinuhjoo,can you share share the evaluation code to me. 
email  1973298281@qq.com

**Comment by jinuhjoo** [comment]
@zhaoweicai 
I kind'a got it by playing a little with the evaluate_object.cpp, but to get detection box results in txt per image, I had to modify a little in the run_mscnn_detection.m file.
Is this how you have approached? My e-mail is bongkamuns@gmail.com
Thanks :-)

**Comment by yqyao** [comment]
Could you  share the evaluation code to me? I also have the question as jinuhjoo. My e-mail is yongqiang_yao@163.com,Thanks.

**Comment by zhaoweicai** [comment]
hi @baiyancheng20 @jinuhjoo @jk1323 @yqyao 
The evaluation script is uploaded. Please check `mscnn/examples/kitti_result/` and `README.md` for how to use it. Thanks!
